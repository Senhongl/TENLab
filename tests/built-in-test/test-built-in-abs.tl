a = [-1,2];
print(abs(a));
b = [-1.0,0.0,-2.4];
print(abs(b));