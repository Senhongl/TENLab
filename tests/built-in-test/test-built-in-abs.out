 1
 2
[ CPUIntType{2} ]
 1.0000
 0.0000
 2.4000
[ CPUDoubleType{3} ]